### PR TITLE
enhance: [cherry-pick] add autoindex mapping for binary/sparse datatype

### PR DESCRIPTION
**Pull request by foxspy**
issue: #22837 
pr: #33624

### CODE REVIEW

**Comment by foxspy** [comment]
related: https://github.com/milvus-io/milvus/pull/33624

File: pkg/util/paramtable/autoindex_param.go [hunk 1]
```diff
@@ -71,12 +74,34 @@ func (p *autoIndexConfig) init(base *BaseTable) {
...
+
 	p.PrepareParams = ParamItem{
 		Key:     "autoIndex.params.prepare",
 		Version: "2.3.2",
 	}
 	p.PrepareParams.Init(base.mgr)
 
+	p.LoadAdaptParams = ParamItem{
+		Key:     "autoIndex.params.load",
+		Version: "2.4.5",
```
Comment (chasingegg) [inline pkg/util/paramtable/autoindex_param.go:101]: why need load params? prepare is enough?

**Pull request merged**